pandas/tests/frame/test_operators.py

```diff
@@ -671,6 +671,28 @@ def _test_seq(df, idx_ser, col_ser):
         exp = DataFrame({'col': [False, True, False]})
         assert_frame_equal(result, exp)
 
+    def test_return_dtypes_bool_op_costant(self):
```

Comment:
can you replicate these tests for series as well (or point to that we have equiv tests)